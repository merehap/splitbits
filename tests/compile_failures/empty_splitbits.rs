use splitbits::*;

fn main() {
    splitbits!();
}
