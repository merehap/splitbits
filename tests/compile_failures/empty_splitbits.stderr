error: proc macro panicked
 --> tests/compile_failures/empty_splitbits.rs:4:5
  |
4 |     splitbits!();
  |     ^^^^^^^^^^^^
  |
  = help: message: splitbits must take at least two arguments: an input value then a template. Found:
          ``
